app/helpers/check_answers_helper.rb

```diff
@@ -1,154 +1,254 @@
+      },
+      {
+        field: charge_text,
+        value: session[:construction_cost],
+      },
+    ]
+  end
+
+  def it_service_items
+    [
+      {
+        field: I18n.t("services.it.type", scope: sections_scope),
+        value: render("coronavirus_form/check_your_answers/helpers/list", items: session[:it_services]),
+      },
+      {
+        field: I18n.t("services.it.description", scope: sections_scope),
+        value: session[:it_services_other],
+      },
+      {
+        field: charge_text,
+        value: session[:it_cost],
+      },
+    ]
+  end
+
+  def other_support_items
+    [{
+      field: t("other_support.description", scope: sections_scope),
+      value: session["offer_other_support"],
     }]
   end
 
-  def offer_staff_type
+  def location_items
     [{
-      field: t("coronavirus_form.questions.offer_staff_type.title"),
-      value: sanitize(Array(session[:offer_staff_type]).flatten.to_sentence),
-      edit: {
-        href: "offer-staff-type?change-answer",
-      },
-    },
-     {
-       field: t("coronavirus_form.questions.offer_staff_type.offer_staff_description.label"),
-       value: sanitize(session[:offer_staff_description]),
-       edit: {
-         href: "offer-staff-type?change-answer",
-       },
-     }]
-  end
-
-  def concat_answer(answer, question)
-    return unless answer
-    return answer if answer.is_a?(String)
-    return answer.flatten.to_sentence if answer.is_a?(Array)
-
-    concatenated_answer = []
-    joiner = " "
-
-    if question.eql?("contact_details")
-      concatenated_answer << "Name: #{answer[:contact_name]}" if answer[:contact_name]
-      concatenated_answer << "Role: #{answer[:role]}" if answer[:role]
-      concatenated_answer << "Phone number: #{answer[:phone_number]}" if answer[:phone_number]
-      concatenated_answer << "Email: #{answer[:email]}" if answer[:email]
-      joiner = "<br>"
-    elsif question.eql?("business_details")
-      concatenated_answer << "Company name: #{answer[:company_name]}" if answer[:company_name]
-      concatenated_answer << "Company registered in the UK: #{answer[:company_is_uk_registered]}"
-      concatenated_answer << "Company number: #{answer[:company_number]}" if answer[:company_number].present?
-      concatenated_answer << "Company size number: #{answer[:company_size]}" if answer[:company_size]
-      concatenated_answer << "Company location: #{answer[:company_location]}" if answer[:company_location]
-      concatenated_answer << "Company postcode: #{answer[:company_postcode]}" if answer[:company_postcode]
-      joiner = "<br>"
+      field: t("location.description", scope: sections_scope),
+      value: render("coronavirus_form/check_your_answers/helpers/list", items: session["location"]),
+    }]
+  end
+
+  def business_detail_items
+    [
+      {
+        field: I18n.t("business_details.name", scope: sections_scope),
+        value: session[:business_details][:company_name],
```

Comment:
I'd suggest using `dig` for getting values within a nested hash, as it handles the absence of the higher level elements more neatly. So here for example:
```ruby
value: session.dig(:business_details, :company_name),
```
I've highlighted this occurrence, but the same pattern appears a lot in this PR.

Reply:
Good spot @reggieb 👍